galley/pkg/config/analysis/analyzers/testdata/virtualservice_host_not_found_gateway.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: gw
+  namespace: istio-system
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - hosts:
+    - foo.com
+    port:
+      name: http
+      number: 80
+      protocol: HTTP
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings-service
+  namespace: default
+spec:
+  gateways:
+  - istio-system/gw
+  hosts:
+  - foo.com # Expected: no validation error since this host exists
+  http:
+  - match:
+    - uri:
+        prefix: /
+  - route:
+    - destination:
+        host: ratings
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings-wrong-host-service
+  namespace: default
+spec:
+  gateways:
+  - istio-system/gw
```

Comment:
can you add a case where we have two gateways defined and its present in one of them?

Reply:
I am not sure about your question, adding 2 more examples there. can you check?